get-started/codelab/index.md

```diff
@@ -0,0 +1,1301 @@
+
+Add the icon and its acton to the build method:
+
+<!-- skip -->
+{% prettify dart %}
+class RandomWordsState extends State<RandomWords> {
+  ...
+  @override
+  Widget build(BuildContext context) {
+    return new Scaffold(
+      appBar: new AppBar(
+        title: new Text('Startup Name Generator'),
+        [[highlight]]actions: <Widget>[[[/highlight]]
+          [[highlight]]new IconButton(icon: new Icon(Icons.list), onPressed: _pushSaved),[[/highlight]]
+        [[highlight]]],[[/highlight]]
+      ),
+      body: _buildSuggestions(),
+    );
+  }
+  ...
+}
+{% endprettify %}
+</li>
+
+<li markdown="1"> Add a `_pushSaved()` method to the RandomWordsState class.
+
+<!-- skip -->
+{% prettify dart %}
+class RandomWordsState extends State<RandomWords> {
+  ...
+  void _pushSaved() {
+  }
+}
+{% endprettify %}
+
+Hot reload the app. The list icon appears in the app bar.
+Tapping it does nothing yet, because the `_pushSaved` method is empty.
+</li>
+
+<li markdown="1"> When the user taps the list icon in the app bar,
+    build a route and push it to the Navigator's stack.
+    This action changes the screen to display the new route.
+
+The content for the new page is built in MaterialPageRoute's `builder`
+property, in an anonymous function.
+
+Add the call to Navigator.push, as shown by the highlighted code,
+which pushes the route to the Navigator's stack.
+
+<!-- skip -->
+{% prettify dart %}
+  void _pushSaved() {
+    [[highlight]]Navigator.of(context).push([[/highlight]]
+      [[highlight]]);[[/highlight]]
+  }
+{% endprettify %}
+</li>
+
+<li markdown="1"> Add the MaterialPageRoute and its builder. For now,
+    add the code that generates the ListTile rows. The `divideTiles()`
+    method of ListTile adds horizontal spacing between each ListTile.
+    The `divided` variable holds the final rows, converted to a list
+    by the convienice function, `toList()`.
+
+<!-- skip -->
+{% prettify dart %}
+  void _pushSaved() {
+    Navigator.of(context).push(
+      [[highlight]]new MaterialPageRoute([[/highlight]]
+        [[highlight]]builder: (context) {[[/highlight]]
+          [[highlight]]final tiles = _saved.map([[/highlight]]
+                [[highlight]](pair) {[[/highlight]]
+              [[highlight]]return new ListTile([[/highlight]]
+                [[highlight]]title: new Text([[/highlight]]
+                  [[highlight]]pair.asPascalCase,[[/highlight]]
+                  [[highlight]]style: _biggerFont,[[/highlight]]
+                [[highlight]]),[[/highlight]]
+              [[highlight]]);[[/highlight]]
+            [[highlight]]},[[/highlight]]
+          [[highlight]]);[[/highlight]]
+          [[highlight]]final divided = ListTile[[/highlight]]
+              [[highlight]].divideTiles([[/highlight]]
+            [[highlight]]context: context,[[/highlight]]
+            [[highlight]]tiles: tiles,[[/highlight]]
+          [[highlight]])[[/highlight]]
+              [[highlight]].toList();[[/highlight]]
+    );
```

Comment:
You're missing a couple closing brackets/parens on this method here. They show up in lines 1098 and 1099 without ever getting a highlighted "this is where you add these" step.

Reply:
Thanks for catching!